dnstable_convert.c

```diff
@@ -963,6 +988,13 @@ main(int argc, char **argv)
 		case 'p':
 			preserve_empty = true;
 			break;
+		case 'S':
+			store_nmsg_source = false;
```

Comment:
Please reverse the sense of `-S` here, so `dnstable_convert` defaults to its old behavior.